staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -1646,6 +1646,11 @@ func (c *Cloud) InstanceShutdownByProviderID(ctx context.Context, providerID str
 	return false, nil
 }
 
+// InstanceMetadataByProviderID returns metadata of the specified instance.
+func (c *Cloud) InstanceMetadataByProviderID(ctx context.Context, providerID, nodeName string) (*cloudprovider.InstanceMetadata, error) {
+	return nil, nil
```

Comment:
Return "unimplemented error" for now.

Same in other places.

Reply:
Done.